snowfakery/standard_plugins/statistical_distributions.py

```diff
@@ -0,0 +1,29 @@
+from numpy.random import normal, lognormal, binomial, exponential, poisson, gamma
+from numpy.random import seed
+
+
+from snowfakery.plugins import SnowfakeryPlugin
+
+
+def wrap(distribution):
+    def _do_distribution(self, **params):
+        random_seed = params.pop("seed", None)
+        seed(random_seed)
+        return float(distribution(**params, size=1).astype(float)[0])
+        # except (TypeError, DataGenValueError, DataGenError):
+        #     raise DataGenError(
+        #         f"Incorrect parameters {params} for {distribution.__name__} distribution.",
+        #         None, None,
+        #     )
```

Comment:
Could you add a docstring to explain what the wrapper does, and remove commented code?